src/burst2safe/safe.py

```diff
@@ -80,6 +82,113 @@ def get_support_dir(self) -> Path:
 
         return data_dir / f'support_{support_version}'
 
+    @staticmethod
```

Comment:
Maybe count_components_and_holes should go in `utils.py` instead of being included in the Safe class. It would be great to call it without having to create a Safe object. It could be called externally for a quick topological check of a burst collection without any specific scenes in mind. It simply checks if the collection of bursts contains multiple components or holes.